src/main/java/de/hpi/bpt/argos/storage/dataModel/event/type/StatusUpdatedEventType.java

```diff
@@ -0,0 +1,228 @@
+		typeAttributes.add(createTypeAttribute(ENTITY_ID_ATTRIBUTE_NAME, statusUpdatedEventType));
+
+		PersistenceAdapterImpl.getInstance().saveArtifacts(typeAttributes.toArray(new TypeAttribute[typeAttributes.size()]));
+
+		statusUpdatedEventType.setTimeStampAttributeId(timestampAttribute.getId());
+		PersistenceAdapterImpl.getInstance().saveArtifacts(statusUpdatedEventType);
+
+		return statusUpdatedEventType;
+	}
+
+	/**
+	 * This method returns a new typeAttribute for a given eventType.
+	 * @param name - the name of the typeAttribute to create
+	 * @param eventType - the eventType this attribute is for
+	 * @return - the newly created typeAttribute
+	 */
+	private static TypeAttribute createTypeAttribute(String name, EventType eventType) {
+		TypeAttribute typeAttribute = new TypeAttributeImpl();
+		typeAttribute.setName(name);
+		typeAttribute.setTypeId(eventType.getId());
+		return typeAttribute;
+	}
+
+	/**
+	 * This method registers a new eventEntityMapping in the given argos instance.
+	 * @param argos - the argos instance to use for mapping
+	 */
+	private static void registerCustomMapping(Argos argos) {
+		EventCreationObserver customMapping =
+				(EventEntityMappingStatus mappingStatus,
+				 EventType eventType,
+				 List<TypeAttribute> eventTypeAttributes,
+				 Event event,
+				 List<Attribute> eventAttributes) -> {
+
+				if (mappingStatus.isMapped() || eventType.getId() != getInstance().getId()) {
+					return;
+				}
+
+				long entityId = getAsLong(getAttribute(ENTITY_ID_ATTRIBUTE_NAME, eventTypeAttributes, eventAttributes));
+
+				if (entityId == 0) {
+					return;
+				}
+
+				Attribute newStatusAttribute = getAttribute(NEW_STATUS_ATTRIBUTE_NAME, eventTypeAttributes, eventAttributes);
+
+				if (newStatusAttribute == null) {
+					return;
+				}
+
+				Entity entity = PersistenceAdapterImpl.getInstance().getEntity(entityId);
+				mappingStatus.setEventOwner(entity);
+				mappingStatus.getStatusUpdateStatus().setNewStatus(newStatusAttribute.getValue());
+			};
+
+		try {
+			argos.addEventEntityMapper(customMapping);
+		} catch (ArgosNotRunningException e) {
+			LoggerUtilImpl.getInstance().error(logger, "cannot register custom mapping for statusUpdatedEventType", e);
+		}
+	}
+
+	/**
+	 * This method searches for a specific attribute, identified by the name of the typeAttribute.
+	 * @param attributeName - the name of the searched attribute
+	 * @param typeAttributes - a list of all typeAttributes
+	 * @param attributes - a list of all attributes
+	 * @return - the attribute of null
+	 */
+	private static Attribute getAttribute(String attributeName, List<TypeAttribute> typeAttributes, List<Attribute> attributes) {
+		for (TypeAttribute typeAttribute : typeAttributes) {
+			if (!typeAttribute.getName().equals(attributeName)) {
+				continue;
+			}
+
+			for (Attribute attribute : attributes) {
+				if (attribute.getTypeAttributeId() == typeAttribute.getId()) {
```

Comment:
Isn't it possible that multiple attributes correspond to this special type attirbtue? Then you would return only the first one you find.